config/clients/java/template/Java/build.gradle.mustache

```diff
@@ -0,0 +1,146 @@
+apply plugin: 'idea'
+apply plugin: 'eclipse'
```

Comment:
TODO: Upgrade Gradle, this looks pretty old... (maybe just the syntax though?)

Add linting (checkstyle) and static analysis (pmd/spotbugs)

Reply:
This template file can be removed -- It gets clobbered by the build.gradle from libraries/native/build.gradle.mustache